app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,333 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
+{
+
+    /**
+     * @var $model
+     */
+    protected $model;
+    /**
+     * EloquentRepository constructor.
+     */
+    public function __construct()
+    {
+        $this->setModel();
+    }
+
+    /**
+     * set reponsitory instant
+     */
+    public function setModel()
+    {
+        $this->model = app()->make(
+            $this->getModel()
+        );
+    }
+
+    /**
+     * @param  int $id
+     * @return mixed
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 220</br>The first parameter of a multi-line function declaration must be on the line after the opening bracket